src/components/Main/Members/utilities.ts

```diff
@@ -0,0 +1,16 @@
+import { MemberStateData } from '../../../store/types';
+
+export function makeNewRows(start: number, end: number, data: MemberStateData[]) {
```

Comment:
This took me a while to understand. Is this helper function used to update an array of selected row IDs? How about we rename it to `makeNewSelectedRows` or `getUpdatedRowSelection` or `updateRows` or `updateSelectedRows`?
```suggestion
export function updateSelectedRows(start: number, end: number, data: MemberStateData[]) {
```

Reply:
agreed!